### PR TITLE
Feature/hero header m

### PR DESCRIPTION
**Pull request by Mikhaylo-Khokhlov**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -17,13 +17,13 @@
   </head>
   <body>
     <include src="./partials/hero-header.html"></include>
-    <include src="./partials/products.html"></include>
+    <!-- <include src="./partials/products.html"></include>
     <include src="./partials/about.html"></include>
```
Comment (Squidoo22) [inline src/index.html]: Стоит раскоментировать)

**Review by Squidoo22 — DISMISSED** [review]

File: src/partials/hero-header.html [hunk 2]
```diff
@@ -1 +1,51 @@
-<header id="hero-header" class="hero-header">Hero-header</header>
+<header id="hero-header" class="hero-header">
+  <div class="container">
+    <div class="top-panel">
```
Comment (izachok) [inline src/partials/hero-header.html:3]: should it be nav tag?

File: src/partials/hero-header.html [hunk 3]
```diff
@@ -1 +1,51 @@
...
+      <img src="./images/logo/logo.png" width="199" height="50" alt="logo icecream" />
```
Comment (izachok) [inline src/partials/hero-header.html:4]: wrap with link

File: src/partials/hero-header.html [hunk 4]
```diff
@@ -1 +1,51 @@
...
+      <h1 class="title hero__title">
+        ice cream <br />
+        made with <br />
+        <span class="hero__title--accent">passion</span>
+      </h1>
+      <ul class="hero__btn">
+        <li class="hero__item">
+          <a href="#products" class="link hero__link">Products</a>
+        </li>
+        <li class="hero__item hero__item--indent">
```
Comment (izachok) [inline src/partials/hero-header.html:42]: :not(:last-child)

File: src/sass/layouts/_hero-header.scss [hunk 5]
```diff
@@ -1,3 +1,184 @@
...
+  margin-bottom: 137px;
+
+  @include for-tablet-up {
+    padding-top: 37px;
+    margin-bottom: 15px;
+  }
+}
+
+.top-panel__icon-menu {
+  stroke: #d41443;
```
Comment (izachok) [inline src/sass/layouts/_hero-header.scss:25]: could you use css var?

File: src/sass/layouts/_hero-header.scss [hunk 6]
```diff
@@ -1,3 +1,184 @@
...
+}
+
+.top-panel__btn {
+  margin-left: auto;
+  padding: 0;
+  background-color: rgba(255, 184, 202, 0);
```
Comment (izachok) [inline src/sass/layouts/_hero-header.scss:31]: why not to use just transparent?

File: src/sass/layouts/_hero-header.scss [hunk 7]
```diff
@@ -1,3 +1,184 @@
...
+
+.hero__item--indent {
+  margin-right: 0;
+}
+
+.hero__link {
+  padding: 7px 17px;
+
+  border-radius: 20px;
+  font-family: DM Sans;
```
Comment (izachok) [inline src/sass/layouts/_hero-header.scss:87]: check font name

File: src/sass/layouts/_hero-header.scss [hunk 8]
```diff
@@ -1,3 +1,184 @@
...
+  font-size: 8px;
+  line-height: 1.25;
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  background-color: var(--secondary-btn-background-color);
+  color: var(--secondary-text-color);
+
+  transition: color var(--animation-time) var(--animation-function),
```
Comment (izachok) [inline src/sass/layouts/_hero-header.scss:99]: you could write transition: get-transition-prop(color), get-transition-prop(background-color);

File: src/sass/layouts/_hero-header.scss [hunk 9]
```diff
@@ -1,3 +1,184 @@
...
+    font-family: DMSans-Bold;
+    font-size: 16px;
+    line-height: 1.31;
+
+    background-color: var(--secondary-text-color);
+    color: var(--secondary-btn-background-color);
+    fill: currentColor;
+
+    transition: color var(--animation-time) var(--animation-function),
+      bacground-color var(--animation-time) var(--animation-function),
```
Comment (izachok) [inline src/sass/layouts/_hero-header.scss:171]: the same. Change to function or fix misprint in bacground-color

**Review by izachok — CHANGES_REQUESTED** [review]

**Pull request closed without merging**